### DISCUSSION TITLE
unable to copy Nightscout history from Atlas

### DISCUSSION THREAD

**Discussion by TomWood1985** [NightscoutFoundation/xDrip#2575] [Q&A] [answered]
Hello
I would appreciate if anyone is able to offer any assistance in helping me to transfer my old nightscout data from Atlas to my newly opened Google cloud nightscout site using the instructions on this page https://navid200.github.io/xDrip/docs/Nightscout/HerokuGone.html.

I'm getting the following error when I paste the mongo dump string into the terminal (I have replaced my user details with & signs; 
&&&&&&&&@nightscout:~$ cd /tmp
&&&&&&&&@nightscout:/tmp$ mongodump --forceTableScan --uri mongodb+srv://heroku_&&&&:&&&&&@cluster-&&&&.fxsrx.mongodb.net/&&&&
2022-12-21T20:31:20.720+0000    Failed: error getting collections for database `&&&&&`: error running `listCollections`. Database: `&&&&&` Err: user is not allowed to do action [listCollections] on [&&&&&.]

Any ideas with how I can rectify this?

Many thanks for any help
Tom

**Comment by Navid200** [comment, marked as answer]
![Screenshot 2022-12-21 155509](https://user-images.githubusercontent.com/51497406/209001270-b0e3539a-b83f-4f3e-af8d-a0cdf340ef2e.png)


What I have circled in the image above from your post should also start with heroku.  Does it and you just didn't type it?
Or, did you enter something else there?

**Reply by TomWood1985** [reply]
Dear Navid 200, 
tried it with your amendment and it worked! 
Thank you so much for your help and quick response

with very best wishes
Tom
